libraries/default.rb

```diff
@@ -19,20 +19,14 @@ def get_memory_limit
       80
     end
 
-    def install_package(name)
-      #
-      # Yes, I know about the "package" resource, but for some reason using a timeout
-      # with it causes a compile-time error.
-      #
-      # We really want to be able to timeout and retry installs to get faster package
-      # mirrors. This is an annoying quirk with the ubuntu package mirror repos.
-      #
-      execute "install #{name}" do
-        environment 'DEBIAN_FRONTEND' => 'noninteractive'
-        command %Q|apt-get install -y #{name}|
-        retries 5
-        retry_delay 15
-        timeout 180
+    def install_package(names, opts="")
```

Comment:
we call this method with multiple package names in several recipes. Because of a quirk in `yum` we have to install each package with a separate command. If a package is already installed yum will exit with an non-zero status code (failure), so if your command is `yum install -y foo bar baz` and `foo` is already installed, yum will fail and `bar` and `baz` will never get installed.